shared/constants.js

```diff
@@ -25,92 +25,233 @@
-            }
+            },
+            commands: ['create', 'createWithTemplate', 'version']            
         },
         forcedroid: {
             name: 'forcedroid',
-            description: 'command line utility for building Android native mobile applications using Salesforce Mobile SDK',
+            description: 'tool for building Android native mobile applications using Salesforce Mobile SDK',
+            sfdx_topic: 'android',
+            sfdx_description: 'command for building a Android native mobile application using Saleforce Mobile SDK',
             dir: 'android',
             platforms: ['android'],
             toolNames: ['git', 'npm'],
             appTypes: ['native', 'native_kotlin'],
             appTypesToPath: {
                 'native': 'AndroidNativeTemplate',
                 'native_kotlin': 'AndroidNativeKotlinTemplate'
-            }
+            },
+            commands: ['create', 'createWithTemplate', 'version']            
         },
         forcehybrid: {
             name: 'forcehybrid',
-            description: 'command line utility for building hybrid mobile applications using Salesforce Mobile SDK',
+            description: 'tool for building hybrid mobile applications using Salesforce Mobile SDK',
+            sfdx_topic: 'hybrid',
+            sfdx_description: 'command for building an hybrid mobile application using Saleforce Mobile SDK',
             dir: 'hybrid',
             platforms: ['ios', 'android'],
             toolNames: ['git', 'npm', 'cordova'],
             appTypes: ['hybrid_local', 'hybrid_remote'],
             appTypesToPath: {
                 'hybrid_local': 'HybridLocalTemplate',
                 'hybrid_remote': 'HybridRemoteTemplate'
-            }
+            },
+            commands: ['create', 'createWithTemplate', 'version']            
         },
         forcereact: {
             name: 'forcereact',
-            description: 'command line utility for building react native mobile applications using Salesforce Mobile SDK',
+            description: 'tool for building react native mobile applications using Salesforce Mobile SDK',
+            sfdx_topic: 'reactnative',
+            sfdx_description: 'command for building a React Native mobile application using Saleforce Mobile SDK',
             dir: 'react',
             platforms: ['ios', 'android'],
             toolNames: ['git', 'npm', 'pod'],
             appTypes: ['react_native'],
             appTypesToPath: {
                 'react_native': 'ReactNativeTemplate'
-            }
+            },
+            commands: ['create', 'createWithTemplate', 'version']
         }
     },
 
-    tools: {
-        git: {
-            checkCmd: 'git --version',
-            minVersion: '2.13'
+    args: {
+        platform: {
+            name: 'platform',
+            'char': 'p',
+            description: cli => 'Comma separated platforms (' + cli.platforms.join(', ') + ')',
```

Comment:
description is used in forcexxx usage and in sfdx plugin help.
prompt is used in forcexxx when entering the arguments interactively.
validate/error is used by both.